commons/zenoh-protocol/src/proto/msg.rs

```diff
@@ -953,7 +953,7 @@ impl Header for Pull {
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct Query {
     pub key: WireExpr<'static>,
-    pub value_selector: String,
+    pub selector_parameters: String,
```

Comment:
Shouldn't be called simply `parameters`?

Reply:
I agree with Luca, I think it should simply be parameters, with potential parameters being: predicate, projection, arguments, etc. etc.